org.trustedqsl.Tqsl.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>org.trustedqsl.Tqsl</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>tqsl</name>
+ <summary>Sign Amateur Radio log files</summary>
+ <developer_name>ARRL, the national association for Amateur Radio</developer_name>
+ <url type="homepage">http://www.arrl.org</url>
+ <description>
+  <p>
+   The mission of the Trusted QSL Group is to keep
+   eQSL an open and free activity for amateur radio
+   operators by promoting the Digital Signature Standard (DSS).
```

Comment:
```suggestion
   A QSL is a confirmation of contact between two amateur radio stations. The ARRL Logbook of the World project is a database which collects data about contacts between amateur stations (QSOs). This package provides the TQSL program for maintaining your digital certificates for LoTW and signing and uploading QSO log files. 
```

Taken from Debian; sounds nicer.

Reply:
`TQSL is an application used to digitally sign and upload QSOs to the ARRL's Logbook of the World (LoTW) online service, and to manage the Callsign Certificates used when digitally signing.`

From Fedora. :-)